gridData/tests/test_mrc.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import absolute_import, division
+
+import pytest
+
+import numpy as np
+from numpy.testing import (assert_almost_equal,
+                           assert_equal)
+
+from gridData import Grid, mrc
+
+from . import datafiles
+
+@pytest.fixture(scope="module")
+def g1():
+    return Grid(datafiles.CCP4, file_format="MRC")
+
+@pytest.fixture(scope="module")
+def g2():
+    data = mrc.MRC()
+    data.read(datafiles.CCP4)
+    grid, edges = data.histogramdd()
+    return Grid(grid=grid, edges=edges)
+
+def test_ccp4_Grid(g1):
+    _test_ccp4(g1)
+
+def test_ccp4_mrc(g2):
+    _test_ccp4(g2)
+
+def _test_ccp4(g):
+    POINTS = 192
+    assert_equal(g.grid.flat, np.arange(1, POINTS+1))
+    assert_equal(g.grid.size, POINTS)
+    assert_almost_equal(g.delta, [3./4, .5, 2./3])
+    assert_equal(g.origin, np.zeros(3))
+
+
+
+@pytest.fixture(scope="module")
+def ccp4data():
+    return mrc.MRC(datafiles.CCP4_1JZV)
+
+@pytest.mark.parametrize('name,value', [
+    # nc, nr, ns appear to be nx, ny, nz
+    #('nc', 96),
+    #('nr', 76),
+    #('ns', 70),
+    ('mode', 2),
+    #('ncstart', -4),
+    #('nrstart', -23),
+    #('nsstart', 102),
+    # not sure why the following values are different in CCP4.CCP4 and mrc.MRC
+    ##('nx', 84),
+    ##('ny', 84),
+    ##('nz', 160),
+    ('nx', 96),
+    ('ny', 76),
+    ('nz', 70),
+    #('xlen', 45.79999923706055),
+    #('ylen', 45.79999923706055),
+    #('zlen', 89.6500015258789),
+    #('alpha', 90.0),
+    #('beta', 90.0),
+    #('gamma', 90.0),
+    ('mapc', 2),
+    ('mapr', 1),
+    ('maps', 3),
+    #('amin', -0.9930942058563232),
+    #('amax', 9.050403594970703),
+    #('amean', -0.0005801090155728161),
+    ('ispg', 92),
+    ('nsymbt', 640),
+    #('lskflg', 0),
+    #('bsaflag', '@'),
+    #('skwmat', None),
+    #('skwtrn', None),
+    #('endianness', 'little'),
+    #('arms', 0.4034915268421173),
+    ('nlabl', 1),
+    # label exists but contains more byte strings
+    #('label', ' Map from fft                                                                   '),
+    ('cella', np.rec.array((45.8, 45.8, 89.65),
+                        dtype=[('x', '<f4'), ('y', '<f4'), ('z', '<f4')])),
+    ('cellb', np.rec.array((90., 90., 90.),
+                           dtype=[('alpha', '<f4'), ('beta', '<f4'), ('gamma', '<f4')])),
+])
+def test_ccp4_read_header(ccp4data, name, value):
+    if type(value) is float:
+        assert_almost_equal(ccp4data.header[name], value, decimal=6)
```

Comment:
I think the convention is now to use `assert_allclose` instead of `assert_almost_equal`.

I fee like there _should_ be a nicer way to do this comparison, but at the end of the day it works so there's no point trying to find the prettiest answer to this problem.

Reply:
YEs, can be changed, of course. I copied & pasted what we had before.

Reply:
kept it,,, need sleep